test/dub_package/README.md

```diff
@@ -0,0 +1,17 @@
+DMD as a library
+================
+
+The tests in this module are high-level and mostly indented to make sure all
+necessary modules are included in the Dub package.
+
+The tests are executable single-file packages for the D's package manager `dub`
+and can be directly executed:
+
+   ./lexer.d
+
+If you want to see the log output or want to pass additional options, use `--single`:
+
+  dub --single -v lexer.d
```

Comment:
This line and line [10](https://github.com/dlang/dmd/pull/7425/files#diff-16b8627c835f07289426c754bd2472efR10) doesn't have the same indentation. For code blocks I recommend the triple backticks.